### ISSUE TITLE
Tests not working

### ISSUE THREAD

**Issue by mikelolasagasti** [anacrolix/fuse#5]
Doing a simple git clone && go test shows problems:

```
$ git clone https://github.com/anacrolix/fuse
$ cd fuse/
$ go test ./...
go: downloading golang.org/x/sys v0.10.0
go: downloading github.com/anacrolix/envpprof v1.3.0
go: downloading github.com/anacrolix/log v0.14.1
go: downloading github.com/anacrolix/generics v0.0.0-20230113004304-d6428d516633
go: downloading golang.org/x/exp v0.0.0-20220428152302-39d4317da171
# github.com/anacrolix/fuse
./fuse.go:176:42: assignment mismatch: 4 variables but mount returns 2 values
FAIL	github.com/anacrolix/fuse [build failed]
ok  	github.com/anacrolix/fuse/cmd/fuse-abort/internal/mountinfo	0.001s
FAIL	github.com/anacrolix/fuse/fs [build failed]
FAIL	github.com/anacrolix/fuse/fs/bench [build failed]
?   	github.com/anacrolix/fuse/fs/fstestutil/spawntest/httpjson	[no test files]
ok  	github.com/anacrolix/fuse/fs/fstestutil/spawntest	0.002s [no tests to run]
FAIL
```

**Comment by anacrolix** [comment]
Thanks for reporting this, I'm taking a look.

**Comment by anacrolix** [comment]
I've pushed a bunch of fixes for linux to master. Sorry the state of fuse is a bit of a mess, I don't really use it anymore from anacrolix/torrent, and support for fuse on MacOS is getting very poor.

**Comment by mikelolasagasti** [comment]
Thanks @anacrolix 

This request comes from restic requiring it https://github.com/restic/restic/blob/master/go.mod#L9 and found the problem while testing deps. Should I suggest restic to change to another module?

**Comment by anacrolix** [comment]
No, let me know how you go with this. Worse case you stick with v0.2.0, it's pretty solid.

**Comment by mikelolasagasti** [comment]
The recommended way to package go stuff for Fedora is to have all dependencies available as individual package. In the case of Restic means that anacrolix/fuse should be packaged in order to build restic with it. Each package should pass available tests when possible and found that anacrolix/fuse was not passing them.

**Comment by anacrolix** [comment]
You could downgrade anacrolix/fuse for your packaged version of anacrolix/torrent. Would that work?

**Comment by anacrolix** [comment]
I think this is fixed in v0.3.1.
